src/client.rs

```diff
@@ -111,11 +123,24 @@ impl ClientFlow {
             handle_generator: HANDLE_GENERATOR_GENERATOR.generate(),
             send_command_state,
             receive_response_state,
+            idle_timeout: None,
         };
 
         Ok((client_flow, greeting))
     }
 
+    pub fn set_some_idle_timeout(&mut self, secs: Option<u64>) {
+        self.idle_timeout = secs;
+    }
+
+    pub fn set_idle_timeout(&mut self, secs: u64) {
+        self.set_some_idle_timeout(Some(secs));
+    }
```

Comment:
Is it necessary to set the timeout after constructing `ClientFlow`? Otherwise I would prefer to move it to `ClientFlowOptions`.

Reply:
I did not dare to change this options, as I was not sure about its scope. So it can definitely move there!